pkg/apis/duck/v1alpha1/subscribable_types.go

```diff
@@ -49,12 +49,40 @@ type SubscriberSpec struct {
 	// UID is used to understand the origin of the subscriber.
 	// +optional
 	UID types.UID `json:"uid,omitempty"`
+	// Generation of the origin of the subscriber with uid:UID.
```

Comment:
Why do we need this? As in, why not just use the observedgeneration below?

Reply:
Do you mean make this observedgeneration rather than generation? Since this is part of Spec, I have kept it as Generation. Channel controllers will use this as a spec and then "do something" based on which they will update the subscribablestatus with observedgeneration. Channels never read the object from apiserver and rely on the spec alone. 
Is that what you asked or I didn't understand the question :) ?